File: 2013-06-07/python/exercise1.py
from pyplasm import *
import random

domainTerrain =  PROD([INTERVALS(10)(64),INTERVALS(10)(64)])

def mounts (dom):
	x = dom[0]
	y = dom[1]
	z = SIN(x)*COS(y)
	dz = x*y/50*random.random()
	return [x, y, dz*z]

model = MAP(mounts)(domainTerrain)

VIEW(model)

File: 2013-06-07/javascript/exercise1.js
var domainTerrain = DOMAIN([[0, 10], [0, 10]])([64, 64]);

var mounts = function(dom){
	var x = dom[0];
	var y = dom[1];
	var z = SIN(x)*COS(y);
	var dz = x*y/50*Math.random();
	return [x, y, dz*z];
}

var model = MAP(mounts)(domainTerrain);

DRAW(model);
